[UPDATE BONUS GUIDE] LNDg - v1.6.4 (#1256)
raspibolt/raspibolt: guide/bonus/lightning/lndg.md

  $ ln -s /data/lnd /home/lndg/.lnd
  ```

> * Check the version number of the latest LNDg release. You can also confirm with the [release page](https://github.com/cryptosharks131/lndg/releases).

Reviewer: ```suggestion
* Check the version number of the latest LNDg release (you can also confirm with the [release page](https://github.com/cryptosharks131/lndg/releases)) and clone it.
```

  ```sh
  $ LATEST_RELEASE=$(wget -qO- https://api.github.com/repos/cryptosharks131/lndg/releases/latest | grep -oP '"tag_name":\s*"\K([^"]+)')
  $ echo $LATEST_RELEASE
  > v1.6.3

>   ```sh

Reviewer: ```suggestion

```

> * Clone the latest release of the project GitHub repository and enter it

Reviewer: ```suggestion
* Check the version number of the latest LNDg release. You can also confirm with the [release page](https://github.com/cryptosharks131/lndg/releases).

  ```sh
  $ LATEST_RELEASE=$(wget -qO- https://api.github.com/repos/cryptosharks131/lndg/releases/latest | grep -oP '"tag_name":\s*"\K([^"]+)')
  $ echo $LATEST_RELEASE
  > v1.6.3
  ```

* Clone the release of the project GitHub repository and enter it.

  {: .highlight }
  > You can also use the latest release (`$LATEST_RELEASE`) if it is higher than the one specified below. However, please be aware that newer releases might not have been thoroughly tested with the rest of the RaspiBolt configuration.

```

### Web server configuration

* As user `lndg` install uwsgi within the LNDg Python virtual environment

  ```sh

>   $ sudo su - lndg

Reviewer: ```suggestion
  $ sudo su - lndg
   cd ~/lndg
```
